packages/ui-react/src/components/IconButton/IconButton.tsx

```diff
@@ -1,22 +1,23 @@
 import classNames from 'classnames';
-import React from 'react';
+import React, { AriaAttributes } from 'react';
 
 import styles from './IconButton.module.scss';
 
 type Props = {
   onClick?: () => void;
   children: JSX.Element;
-  'aria-label': string;
   tabIndex?: number;
   className?: string;
+  'aria-label'?: AriaAttributes['aria-label'];
+  'aria-controls'?: AriaAttributes['aria-controls'];
 };
 
-const IconButton: React.FC<Props> = ({ children, onClick, 'aria-label': ariaLabel, tabIndex = 0, className }: Props) => {
+const IconButton: React.FC<Props> = ({ children, onClick, tabIndex = 0, className, ...rest }: Props) => {
   return (
     <div
+      {...rest}
```

Comment:
I think we better spread the rest property after assigning all the other properties, so that we don't override props (such as `tabIndex`) after it, and we have full control from above.